apps/image-editor/src/js/action.js

```diff
@@ -405,6 +413,88 @@ export default {
     );
   },
 
+  /**
+   * Resize Action
+   * @returns {Object} actions for ui resize
+   * @private
+   */
+  _resizeAction() {
+    return extend(
+      {
+        getCurrentDimensions: () => {
+          return this._graphics.getComponent(componentNames.RESIZE).getCurrentDimensions();
```

Comment:
You are using the components of graphics.
Could you create API for graphics?
like this...
```js
return this._graphics.getCurrentDimensions();
```
The other parts are the same.